### PR TITLE
Editorial pass on JEA content

### PR DESCRIPTION
**Pull request by sdwheeler**
Preparing to move content into main docset.

Related to [AB#1568679](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1568679)

File / Acrolinx   score | Old | New
-- | -- | --
audit-and-report.md | 81 | 95
overview.md | 79 | 88
prerequisites.md | 78 | 96
register-jea.md | 88 | 96
role-capabilities.md | 82 | 91
security-considerations.md | 78 | 89
session-configurations.md | 77 | 90
using-jea.md | 81 | 91

### CODE REVIEW

File: jea/audit-and-report.md [hunk 1]
```diff
@@ -134,14 +160,14 @@ PS>CommandInvocation(Get-Service): "Get-Service"
...
-The InputObject of Out-Default is the PowerShell object returned from the command.
-The details of that object are printed a few lines below, closely mimicking what the user would have seen.
+The output of each command also triggers a **CommandInvocation**, usually to `Out-Default`. The
+**InputObject** of `Out-Default` is the PowerShell object returned from the command. The details of
+that object are printed a few lines below, closely mimicking what the user would have seen.
 
 ## See also
 
-- [*PowerShell ♥ the Blue Team* blog post on security](https://blogs.msdn.microsoft.com/powershell/2015/06/09/powershell-the-blue-team/)
+[*PowerShell ♥ the Blue Team* blog post on security](https://blogs.msdn.microsoft.com/powershell/2015/06/09/powershell-the-blue-team/)
```
Comment (davidsmatlak) [inline jea/audit-and-report.md]: This link resolves to https://devblogs.microsoft.com/powershell/powershell-the-blue-team/

File: jea/overview.md [hunk 2]
```diff
@@ -1,49 +1,50 @@
...
+Highly privileged accounts used to administer your servers pose a serious security risk. Should an
+attacker compromise one of these accounts, they could launch [lateral attacks](http://aka.ms/pth)
+across your organization. Each compromised account gives an attacker access to even more accounts
+and resources, and puts them one step closer to stealing company secrets, launching a
+denial-of-service attack, and more.
+
+It's not always easy to remove administrative privileges, either. Consider the common scenario where
+the DNS role is installed on the same machine as your Active Directory Domain Controller. Your DNS
+administrators require local administrator privileges to fix issues with the DNS server. But to do
+so, you must make them members of the highly privileged "Domain Admins" security group. This
```
Comment (davidsmatlak) [inline jea/overview.md]: Remove the quotes around Domain Admins and use bold.

File: jea/overview.md [hunk 3]
```diff
@@ -1,49 +1,50 @@
...
+approach effectively gives DNS Administrators control over your whole domain and access to all
+resources on that machine.
+
+JEA addresses this problem through the principle of *Least Privilege*. With JEA, you can configure a
```
Comment (davidsmatlak) [inline jea/overview.md]: Use bold for Least Privilege.

File: jea/overview.md [hunk 4]
```diff
@@ -1,49 +1,50 @@
...
+management endpoint for DNS administrators that gives them access only to the PowerShell commands
+they need to get their job done. This means you can provide the appropriate access to repair a
+poisoned DNS cache or restart the DNS server without unintentionally giving them rights to Active
+Directory, or to browse the file system, or run potentially dangerous scripts. Better yet, when the
+JEA session is configured to use temporary privileged virtual accounts, your DNS administrators can
+connect to the server using *non-admin* credentials and still run commands that typically require
```
Comment (davidsmatlak) [inline jea/overview.md]: Use bold for non-admin

File: jea/overview.md [hunk 5]
```diff
@@ -1,49 +1,50 @@
...
+  service accounts to perform privileged actions on behalf of regular users.
+- **Limit what users can do** by specifying which cmdlets, functions, and external commands they can
+  run.
+- **Better understand what your users are doing** with transcripts and logs that show you exactly
+  which commands a user executed during their session.
+
+**Why is JEA important?**
+
+Highly privileged accounts used to administer your servers pose a serious security risk. Should an
+attacker compromise one of these accounts, they could launch [lateral attacks](http://aka.ms/pth)
```
Comment (davidsmatlak) [inline jea/overview.md]: Use the https link that the aka link resolves to.

File: jea/prerequisites.md [hunk 6]
```diff
@@ -1,99 +1,108 @@
...
-Windows 10 users should install the latest feature updates to obtain the current version of Windows PowerShell.
+Windows 10 users should install the latest feature updates to obtain the current version of Windows
+PowerShell.
 
 ## Enable PowerShell Remoting
 
-PowerShell Remoting provides the foundation on which JEA is built.
-It is therefore necessary to ensure PowerShell Remoting is enabled and [properly secured](/powershell/scripting/setup/winrmsecurity) on your system before you can use JEA.
+PowerShell Remoting provides the foundation on which JEA is built. It's necessary to ensure
+PowerShell Remoting is enabled and [properly secured](/powershell/scripting/learn/remoting/winrmsecurity)
```
Comment (davidsmatlak) [inline jea/prerequisites.md]: link text WinRM Security

File: jea/register-jea.md [hunk 7]
```diff
@@ -1,84 +1,98 @@
...
+- You've selected a name for your JEA endpoint.
 
-You also need to select a name for your JEA endpoint.
-The name of the JEA endpoint is required when users want to connect to the system using JEA.
-You can use the [Get-PSSessionConfiguration](https://msdn.microsoft.com/powershell/reference/5.1/microsoft.powershell.core/get-pssessionconfiguration) cmdlet to check the names of existing endpoints on the system.
-Endpoints that start with 'microsoft' are typically shipped with Windows.
-The 'microsoft.powershell' endpoint is the default endpoint used when connecting to a remote PowerShell endpoint.
+The name of the JEA endpoint is required when users connect to the system using JEA. The
+[Get-PSSessionConfiguration](/powershell/module/microsoft.powershell.core/get-pssessionconfiguration)
+cmdlet lists the names of the endpoints on a system. Endpoints that start with 'microsoft' are
```
Comment (davidsmatlak) [inline jea/register-jea.md]: Use backticks around microsoft.

File: jea/role-capabilities.md [hunk 8]
```diff
@@ -84,69 +100,81 @@ VisibleCmdlets = @{ Name = 'Restart-Service'; Parameters = @{ Name = 'Name'; Val
...
-> For best security practices, it is not recommended to use wildcards when defining visible cmdlets or functions.
-> Instead, you should explicitly list each trusted command to ensure no other commands that share the same naming scheme are unintentionally authorized.
+> For best security practices, it is not recommended to use wildcards when defining visible cmdlets
+> or functions. Instead, you should explicitly list each trusted command to ensure no other commands
+> that share the same naming scheme are unintentionally authorized.
 
-You cannot apply both a ValidatePattern and ValidateSet to the same cmdlet or function.
+You can't apply both a **ValidatePattern** and **ValidateSet** to the same cmdlet or function.
 
 If you do, the ValidatePattern will override the ValidateSet.
```
Comment (davidsmatlak) [inline jea/role-capabilities.md]: bold ValidatePattern

File: jea/role-capabilities.md [hunk 9]
```diff
@@ -157,42 +185,56 @@ FunctionDefinitions = @{
...
+includes the proxy functions JEA creates to constrain existing commands.
 
-Select-Object is a default, constrained cmdlet in all JEA sessions that doesn't allow you to select arbitrary properties on objects.
-To use the unconstrained Select-Object in functions, you must explicitly request the full implementation by specifying the FQMN.
-Any constrained cmdlet in a JEA session will exhibit the same behavior when invoked from a function, in line with PowerShell's [command precedence](/powershell/module/microsoft.powershell.core/about/about_command_precedence).
+By default, `Select-Object` is a constrained cmdlet in all JEA sessions that doesn't allow the
+selection of arbitrary properties on objects. To use the unconstrained `Select-Object` in functions,
+you must explicitly request the full implementation using the FQMN. Any constrained cmdlet in a JEA
+session has the same constraints when invoked from a function. For more information, see
+PowerShell's [command precedence](/powershell/module/microsoft.powershell.core/about/about_command_precedence).
```
Comment (davidsmatlak) [inline jea/role-capabilities.md]: link text: about_Command_Precedence

File: jea/role-capabilities.md [hunk 10]
```diff
@@ -157,42 +185,56 @@ FunctionDefinitions = @{
...
 
-If you are writing a lot of custom functions, it may be easier to put them in a [PowerShell Script Module](/powershell/developer/windows-powershell).
-You can then make those functions visible in the JEA session using the VisibleFunctions field like you would with built-in and third party modules.
+If you're writing several custom functions, it's simpler to put them in a [PowerShell Script Module](/powershell/developer/windows-powershell).
```
Comment (davidsmatlak) [inline jea/role-capabilities.md]: Link text: Windows PowerShell SDK

File: jea/role-capabilities.md [hunk 11]
```diff
@@ -157,42 +185,56 @@ FunctionDefinitions = @{
...
 ## Place role capabilities in a module
 
-In order for PowerShell to find a role capability file, it must be stored in a "RoleCapabilities" folder in a PowerShell module.
-The module can be stored in any folder included in the `$env:PSModulePath` environment variable, however you should not place it in System32 (reserved for built-in modules) or a folder where the untrusted, connecting users could modify the files.
-Below is an example of creating a basic PowerShell script module called *ContosoJEA* in the "Program Files" path.
+In order for PowerShell to find a role capability file, it must be stored in a "RoleCapabilities"
+folder in a PowerShell module. The module can be stored in any folder included in the
+`$env:PSModulePath` environment variable, however you shouldn't place it in System32 (reserved for
+built-in modules) or a folder where the untrusted, connecting users could modify the files. Below is
+an example of creating a basic PowerShell script module called *ContosoJEA* in the "Program Files"
```
Comment (davidsmatlak) [inline jea/role-capabilities.md]: Remove italics and double-quotes.

File: jea/role-capabilities.md [hunk 12]
```diff
@@ -202,71 +244,95 @@ New-Item -ItemType Directory $rcFolder
...
-For administrators looking to lock down access to the role capabilities, ensure Local System has read access to the role capability files and containing modules.
+For administrators looking to lock down access to the role capabilities, ensure Local System has
+read access to the role capability files and containing modules.
 
 ## How role capabilities are merged
 
-Users can be granted access to multiple role capabilities when they enter a JEA session depending on the role mappings in the [session configuration file](session-configurations.md).
-When this happens, JEA tries to give the user the *most permissive* set of commands allowed by any of the roles.
+Users are granted access to all matching role capabilities in the [session configuration file](session-configurations.md)
+when they enter a JEA session. JEA tries to give the user the *most permissive* set of commands
```
Comment (davidsmatlak) [inline jea/role-capabilities.md]: Remove italics around most permissive.

File: jea/security-considerations.md [hunk 13]
```diff
@@ -1,118 +1,143 @@
...
+| Domain controller            | Default                             | Domain user, member of '*DOMAIN*\Domain Admins'         | Computer account     |
+| Domain controller            | Domain groups A and B               | Domain user, member of '*DOMAIN*\A', '*DOMAIN*\B'       | Computer account     |
+| Member server or workstation | Default                             | Local user, member of '*BUILTIN*\Administrators'        | Computer account     |
+| Member server or workstation | Local groups C and D                | Local user, member of '*COMPUTER*\C' and '*COMPUTER*\D' | Computer account     |
+
+When you look at security audit events and application event logs, you see that each JEA user
+session has a unique virtual account. This unique account helps you track user actions in a JEA
+endpoint back to the original user who ran the command. Virtual account names follow the format
+`WinRM Virtual Users\WinRM_VA_<ACCOUNTNUMBER>_<DOMAIN>_<sAMAccountName>` For example, if user
+"Alice" in domain "Contoso" restarts a service in a JEA endpoint, the username associated with any
```
Comment (davidsmatlak) [inline jea/security-considerations.md]: remove quotes around Alice and Contoso

File: jea/security-considerations.md [hunk 14]
```diff
@@ -1,118 +1,143 @@
...
+service control manager events would be `WinRM Virtual Users\WinRM_VA_1_contoso_alice`.
+
+**Group-managed service accounts (gMSAs)** are useful when a member server needs to have access to
+network resources in the JEA session. For example, when a JEA endpoint is used to control access to
+a REST API service hosted on a different machine. It's easy to write functions to invoke the REST
+APIs, but you need a network identity to authenticate with the API. Using a group-managed service
+account makes the "second hop" possible while maintaining control over which computers can use the
```
Comment (davidsmatlak) [inline jea/security-considerations.md]: remove quotes around second hop

File: jea/security-considerations.md [hunk 15]
```diff
@@ -135,16 +163,21 @@ A more secure version of this same role capability would look like:
...
-## JEA does not protect against admins
+## JEA doesn't protect against admins
 
-One of the core principles of JEA is that it allows non-admins to perform *some* admin tasks.
-JEA does not protect against those who already have administrator privileges.
-Users who belong "domain admins," "local admins," or other highly privileged groups in your environment will still be able to get around JEA's protections by signing into the machine via another means.
-They could, for example, sign in with RDP, use remote MMC consoles, or connect to unconstrained PowerShell endpoints.
-Local admins on the system can also modify JEA configurations to allow additional users to manage the system or change a role capability to extend the scope of what a user can do in their JEA session.
-It is therefore important to evaluate your JEA users' extended permissions to see if there are other ways they could gain privileged access to the system.
+One of the core principles of JEA is that it allows non-admins to do *some* admin tasks. JEA doesn't
```
Comment (davidsmatlak) [inline jea/security-considerations.md]: remove italics from the word: some

File: jea/security-considerations.md [hunk 16]
```diff
@@ -135,16 +163,21 @@ A more secure version of this same role capability would look like:
...
+protect against users who already have administrator privileges. Users who belong "domain admins",
```
Comment (davidsmatlak) [inline jea/security-considerations.md]: remove quotes around domain admins, local admins, and just-in-time

File: jea/session-configurations.md [hunk 17]
```diff
@@ -77,81 +83,100 @@ RunAsVirtualAccountGroups = 'NetworkOperator', 'NetworkAuditor'
...
+
+For more information about PSDrives, see [Managing PowerShell drives](/powershell/scripting/samples/managing-windows-powershell-drives).
 
 ### Role definitions
 
-Role definitions in a session configuration file define the mapping of *users* to *roles*.
-Every user or group included in this field will automatically be granted permission to the JEA endpoint when it is registered.
-Each user or group can be included as a key in the hashtable only once, but can be assigned multiple roles.
-The name of the role capability should be the name of the role capability file, without the .psrc extension.
+Role definitions in a session configuration file define the mapping of *users* to *roles*. Every
```
Comment (davidsmatlak) [inline jea/session-configurations.md]: Use bold for users and roles

File: jea/session-configurations.md [hunk 18]
```diff
@@ -161,11 +186,13 @@ RoleDefinitions = @{
...
-If a user belongs to more than one group in the role definition, they will get access to the roles of each.
-If two roles grant access to the same cmdlets, the most permissive parameter set will be granted to the user.
+If a user belongs to more than one group in the role definition, they get access to the roles of
+each. When two roles grant access to the same cmdlets, the most permissive parameter set is granted to
+the user.
 
-When specifying local users or groups in the role definitions field, be sure to use the computer name (not *localhost* or *.*) before the backslash.
-You can check the computer name by inspecting the `$env:computername` variable.
+When specifying local users or groups in the role definitions field, be sure to use the computer
+name, not *localhost* or wildcards. You can check the computer name by inspecting the
```
Comment (davidsmatlak) [inline jea/session-configurations.md]: use bold for Localhost

File: jea/session-configurations.md [hunk 19]
```diff
@@ -175,27 +202,34 @@ RoleDefinitions = @{
...
 
 ### Conditional access rules
 
-All users and groups included in the RoleDefinitions field are automatically granted access to JEA endpoints.
-Conditional access rules allow you to refine this access and require users to belong to additional security groups which do not impact the roles which they are assigned.
-This can be useful if you want to integrate a "just in time" privileged access management solution, smartcard authentication, or other multifactor authentication solution with JEA.
+All users and groups included in the **RoleDefinitions** field are automatically granted access to
+JEA endpoints. Conditional access rules allow you to refine this access and require users to belong
+to additional security groups that don't impact the roles to which they're assigned. This is useful
+when you want to integrate a "just-in-time" privileged access management solution, smartcard
```
Comment (davidsmatlak) [inline jea/session-configurations.md]: remove quotes from just-in-time

File: jea/session-configurations.md [hunk 20]
```diff
@@ -210,25 +244,28 @@ RequiredGroups = @{ And = 'elevated-jea', @{ Or = '2FA-logon', 'smartcard-logon'
...
+to specific cmdlets, functions, or providers, you can do so right in the session configuration file.
+For a full list of supported properties in the session configuration file, run
+`Get-Help New-PSSessionConfigurationFile -Full`.
 
 ## Testing a session configuration file
 
-You can test a session configuration using the [Test-PSSessionConfigurationFile](https://msdn.microsoft.com/powershell/reference/5.1/microsoft.powershell.core/test-pssessionconfigurationfile) cmdlet.
-It is strongly recommended that you test your session configuration file if you have edited the pssc file manually using a text editor to ensure the syntax is correct.
-If a session configuration file does not pass this test, it will not be able to be successfully registered on the system.
+You can test a session configuration using the [Test-PSSessionConfigurationFile](/powershell/microsoft.powershell.core/test-pssessionconfigurationfile)
```
Comment (davidsmatlak) [inline jea/session-configurations.md]: broken link

File: jea/using-jea.md [hunk 21]
```diff
@@ -1,44 +1,49 @@
...
 
-To use JEA interactively, you will need:
-- The name of the computer you are connecting to (can be the local machine)
+- The name of the computer you're connecting to (can be the local machine)
 - The name of the JEA endpoint registered on that computer
-- Credentials for the computer that have access to the JEA endpoint
+- Credentials that have access to the JEA endpoint on that computer
 
-With that information in hand, you can start a JEA session using [New-PSSession](https://msdn.microsoft.com/powershell/reference/5.1/microsoft.powershell.core/New-PSSession) or [Enter-PSSession](https://msdn.microsoft.com/powershell/reference/5.1/microsoft.powershell.core/enter-pssession).
+Given that information, you can start a JEA session using the [New-PSSession](/powershell/microsoft.powershell.core/New-PSSession)
```
Comment (davidsmatlak) [inline jea/using-jea.md]: broken link

File: jea/using-jea.md [hunk 22]
```diff
@@ -100,42 +114,51 @@ $filteredCommands = $commands.Name | Where-Object { $jeaDefaultCmdlets -notconta
...
 ## Using JEA programmatically
 
-JEA can also be used in automation systems and in user applications, such as in-house helpdesk apps and web sites.
-The approach is the same as that for building apps that talk to unconstrained PowerShell endpoints, with the caveat that the program should be aware that JEA is limiting the commands that can be run in the remote session.
+JEA can also be used in automation systems and in user applications, such as in-house helpdesk apps
+and websites. The approach is the same as that for building apps that talk to unconstrained
+PowerShell endpoints. Ensure the program is designed to work with limitation imposed by JEA.
 
-For simple, one-off tasks, you can use [Invoke-Command](https://msdn.microsoft.com/powershell/reference/5.1/microsoft.powershell.core/invoke-command) to run a set of commands using JEA.
+For simple, one-off tasks, you can use [Invoke-Command](/powershell/microsoft.powershell.core/invoke-command)
```
Comment (davidsmatlak) [inline jea/using-jea.md]: broken link

File: jea/audit-and-report.md (hunk 1 above)
Comment (sdwheeler) [reply jea/audit-and-report.md]: fixed

File: jea/overview.md (hunk 5 above)
Comment (sdwheeler) [reply jea/overview.md]: fixed

File: jea/overview.md (hunk 2 above)
Comment (sdwheeler) [reply jea/overview.md]: fixed

File: jea/overview.md (hunk 3 above)
Comment (sdwheeler) [reply jea/overview.md]: fixed

File: jea/overview.md (hunk 4 above)
Comment (sdwheeler) [reply jea/overview.md]: fixed

File: jea/prerequisites.md (hunk 6 above)
Comment (sdwheeler) [reply jea/prerequisites.md]: Fixed

File: jea/register-jea.md (hunk 7 above)
Comment (sdwheeler) [reply jea/register-jea.md]: fixed

File: jea/role-capabilities.md (hunk 8 above)
Comment (sdwheeler) [reply jea/role-capabilities.md]: Fixed

File: jea/role-capabilities.md (hunk 9 above)
Comment (sdwheeler) [reply jea/role-capabilities.md]: fixed

File: jea/role-capabilities.md (hunk 10 above)
Comment (sdwheeler) [reply jea/role-capabilities.md]: I removed the link. I don't think we need it here.

File: jea/role-capabilities.md (hunk 11 above)
Comment (sdwheeler) [reply jea/role-capabilities.md]: Fixed

File: jea/role-capabilities.md (hunk 12 above)
Comment (sdwheeler) [reply jea/role-capabilities.md]: Fixed

File: jea/security-considerations.md (hunk 13 above)
Comment (sdwheeler) [reply jea/security-considerations.md]: fixed

File: jea/security-considerations.md (hunk 14 above)
Comment (sdwheeler) [reply jea/security-considerations.md]: fixed

File: jea/security-considerations.md (hunk 15 above)
Comment (sdwheeler) [reply jea/security-considerations.md]: fixed

File: jea/security-considerations.md (hunk 16 above)
Comment (sdwheeler) [reply jea/security-considerations.md]: fixed

File: jea/session-configurations.md (hunk 17 above)
Comment (sdwheeler) [reply jea/session-configurations.md]: fixed

File: jea/session-configurations.md (hunk 18 above)
Comment (sdwheeler) [reply jea/session-configurations.md]: fixed

File: jea/session-configurations.md (hunk 19 above)
Comment (sdwheeler) [reply jea/session-configurations.md]: fixed

File: jea/session-configurations.md (hunk 20 above)
Comment (sdwheeler) [reply jea/session-configurations.md]: fixed

File: jea/using-jea.md (hunk 21 above)
Comment (sdwheeler) [reply jea/using-jea.md]: Fixed

File: jea/using-jea.md (hunk 22 above)
Comment (sdwheeler) [reply jea/using-jea.md]: fixed

**Pull request merged**